tests/cli/test_run.py

```diff
@@ -395,6 +398,7 @@ def test_successful_main_function_with_config(
     captured = capsys.readouterr()
     lines = captured.out.strip().split("\n")
     expected_lines = [
+        f"Detected Platform: {PLATFORM}",
```

Comment:
Is there a reason you used different code here?

Reply:
None, I'll change it to the same code as the others.